agent-requirements.txt

```diff
@@ -0,0 +1,9 @@
+anyio
```

Comment:
Can we just inline these into `pyproject.toml` as normal `extra`s? Imo we don't need these extra requirements files, they are just distracting